### ISSUE TITLE
Installation Optimise For Chinese Users Who Behind the Wall

### ISSUE THREAD

**Issue by jinfagang** [pytorch/pytorch#4400]
As you know, we are just can not access all Google etc access behind the f*k great wall. The pytorch.org wesite can not load properly when click pip and cuda chips. Just not responding,  I think the site using some access which block in China. Just for suggestion, write more install command in Github README.md or anywhere else.

**Comment by zou3519** [comment]
Interesting. Sure, I think it's reasonable to add some installation instructions to the README. I'll put up a pr for it later.

In the meantime, here's a list of the relevant `whl` files.
```
# py2.7, cuda 8 or no cuda
pip install http://download.pytorch.org/whl/cu80/torch-0.3.0.post4-cp27-cp27mu-linux_x86_64.whl
# py2.7, cuda 9
pip install http://download.pytorch.org/whl/cu90/torch-0.3.0.post4-cp27-cp27mu-linux_x86_64.whl 
# py2.7, cuda 7.5
pip install http://download.pytorch.org/whl/cu75/torch-0.3.0.post4-cp27-cp27mu-linux_x86_64.whl

# python 3.5
pip3 install http://download.pytorch.org/whl/cu80/torch-0.3.0.post4-cp35-cp35m-linux_x86_64.whl 
pip3 install http://download.pytorch.org/whl/cu90/torch-0.3.0.post4-cp35-cp35m-linux_x86_64.whl   
pip3 install http://download.pytorch.org/whl/cu75/torch-0.3.0.post4-cp35-cp35m-linux_x86_64.whl

# python 3.6
pip3 install http://download.pytorch.org/whl/cu80/torch-0.3.0.post4-cp36-cp36m-linux_x86_64.whl 
pip3 install http://download.pytorch.org/whl/cu90/torch-0.3.0.post4-cp36-cp36m-linux_x86_64.whl   
pip3 install http://download.pytorch.org/whl/cu75/torch-0.3.0.post4-cp36-cp36m-linux_x86_64.whl
```

**Comment by hughperkins** [comment]
In my experience/opinion, anything in `.com` is allowed by default, and blacklisted if some reason determined later. Meanwhile, `.org` is blacklisted by default, and whitelisted if some reason determined later. python.org was stuck in the default blacklist for years.... (might still be there).

tldr; if some way of hosting things in a `.com` domain, might fix a bunch of access issues.

**Comment by apaszke** [comment]
@hughperkins that's helpful, we'll try to set up `.com` aliases

**Comment by chenyuntc** [comment]
@apaszke btw, the [forum](https://discuss.pytorch.org) also seems to be blocked from time to time.

**Comment by charlielam0615** [comment]
I haven't been able to open the Pytorch Forum without the aid of VPNs. But now the censorship grips even tighter, the good old days with VPNs are gone. It becomes a serious problem now.

**Comment by chenyuntc** [comment]
As for the official website: pytorch.org , there is a mirror maintained by @wangyangting  http://pytorch.apachecn.org  including pytorch docs translation and tutorials translation (on the way)
